[{"ContainerToolsException.Message":"Container BooksAPP  Removing","ContainerToolsException.ErrorCode":"DT1001","ContainerToolsException.StackTrace":"   in System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   in System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   in Microsoft.Docker.DockerComposeClient.<ExecuteAsync>d__13.MoveNext() in C:\\a\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:riga 253\r\n--- Fine traccia dello stack da posizione precedente dove è stata generata l'eccezione ---\r\n   in System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   in System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   in Microsoft.Docker.DockerComposeClient.<DownAsync>d__8.MoveNext() in C:\\a\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:riga 115\r\n--- Fine traccia dello stack da posizione precedente dove è stata generata l'eccezione ---\r\n   in System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   in System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   in Microsoft.Docker.DockerWorkspace.<ComposeDownContainersAsync>d__31.MoveNext() in C:\\a\\1\\s\\src\\Microsoft.Docker\\DockerWorkspace.cs:riga 418\r\n--- Fine traccia dello stack da posizione precedente dove è stata generata l'eccezione ---\r\n   in System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   in System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   in Microsoft.Docker.BuildTasks.DockerBuildTask.<ExecuteAsync>d__38.MoveNext() in C:\\a\\1\\s\\src\\Microsoft.Docker.BuildTasks\\DockerBuildTask.cs:riga 87","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.CommandLine.Program":"docker-compose","ContainerToolsException.CommandLine.TelemetrySafeArguments":"","ContainerToolsException.CommandLine.ExitCode":"-1","ContainerToolsException.CommandLine.ErrorOutput":"Container BooksAPP  Stopping\r\nContainer BooksAPP  Stopping\r\nContainer BooksDB  Stopping\r\nContainer BooksDB  Stopping\r\nContainer BooksDB  Stopped\r\nContainer BooksDB  Removing\r\nContainer BooksAPP  Stopped\r\nContainer BooksAPP  Removing"}]